src/bundle/Resources/views/fieldtypes/preview/content_fields.html.twig

```diff
@@ -69,14 +69,19 @@
     {% if not ez_is_field_empty( content, field ) %}
         {% set field_value = field.value.value|ez_full_datetime %}
         {{ block( 'simple_block_field' ) }}
+        {% if fieldSettings.useSeconds %}
+            <div class="ez-alert ez-alert--info mt-2">
+                {{ 'ezdatetime.useseconds.enabled'|trans()|desc('`The date format is based on your user preferences and does not include seconds even if the field allows it`') }}
+            </div>
+        {% endif %}
     {% endif %}
 {% endspaceless %}
 {% endblock %}
 
 {% block ezdate_field %}
 {% spaceless %}
     {% if not ez_is_field_empty( content, field ) %}
-        {% set field_value = field.value.date|ez_full_date %}
+        {% set field_value = field.value.date|ez_full_date('UTC') %}
```

Comment:
We will be displaying ezdate in UTC, regardless of user time zone from settings?

Reply:
@tischsoic same as in the prev implementation, I do not change this behaviour

Reply:
Now it will be inconsistent with date picked in edit mode considering last changes: https://github.com/ezsystems/ezplatform-admin-ui/pull/893
Or maybe we are going to revert those changes?
ping @SylvainGuittard

Reply:
@webhdx

Reply:
It cannot be `UTC`. The editor will be confused.

Reply:
Date should not be affected by timezone. When you select `22.03.2019` and you change the timezone you still expect to see `22.03.2019`. We had this issue in the past and this is why we want to display this date in UTC as we store timestamp in the database.

Reply:
Same thing for `eztime` - once `15:00` is selected it should be `15:00` even when user changed timezone in the settings. Basically only `ezdatetime` is timezone dependent.

Reply:
I agree with @webhdx especially that this is a fieldtype value not a regular UI date or time

Reply:
@lserwatka @webhdx I understand your point. But the date picker depends on the user preferences and the time zone can be different than UTC. So the date in preview might be different from the date the user selected. Or did I miss something?

Reply:
Let's discuss this in private on Monday.

Reply:
I was thinking about it and it actully makes sense for me to have _eztime_ and _ezdate_ always in UTC.

Reply:
After internal discussion, `ezdate` and `eztime` should use UTC. 
Thanks guys.